### DISCUSSION TITLE
Examples of Flowmap.blue movement maps in pydeck?

### DISCUSSION THREAD

**Discussion by brianmcdonald** [visgl/deck.gl#7434] [Q&A] [answered]
I have used some pydeck examples as a starting point for some maps. They have been really useful. I noticed that there are no flowmap.blue examples. Is it possible to create flowmap.blue movement visualizations using pydeck.gl, rather than needing to use the standalone flowmap.blue website? If so, can you point me to some examples? 

A big thank you to those contributing the project, it's a fantastic tool, I greatly appreciate your work.

**Comment by kylebarron** [comment, marked as answer]
I think you'd need to create a custom pydeck layer to do this, as it probably requires some JS code that doesn't exist by default in pydeck
